virtcontainers/sandbox.go

```diff
@@ -1239,6 +1247,10 @@ func (s *Sandbox) stopVM() error {
 	span, _ := s.trace("stopVM")
 	defer span.Finish()
 
+	if err := s.agent.stopSandbox(s); err != nil {
```

Comment:
This looks good, but please go one step further by actually calling `(s *Sandbox) stopVM()` from `(s *Sandbox) Stop()`.

Reply:
Done as well.